modules/dev-tools/templates/.eslintrc.js

```diff
@@ -0,0 +1,59 @@
+const typescriptConfigs = require('@typescript-eslint/eslint-plugin').configs;
+
+module.exports = {
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 2018
+  },
+  plugins: ['markdown'],
+  extends: [
+    'uber-es2015',
+    'prettier'
+  ],
+  rules: {
+    'guard-for-in': 0,
+    'no-inline-comments': 0,
+    'camelcase': 0
+  },
+  overrides: [
+    {
+      files: ['**/*.ts', '**/*.tsx'],
+      parser: '@typescript-eslint/parser',
+      env: {browser: true, es6: true, node: true},
```

Comment:
```suggestion
      env: {browser: true, es2021: true, node: true},
```
Why are we specifying the `envs` in this nested parserOptions and not up top?

Reply:
Yes, good point. Probably this can be pulled up